### ISSUE TITLE
Is basic functionality available for CPU?

### ISSUE THREAD

**Issue by rafaelorozco** [kailaix/ADSeismic.jl#1]
Hey! I have been trying to run some of the examples and have run into problems because I don't have GPU's on my laptop. Can the software run on CPU's?

**Comment by wayneweiqiang** [comment]
Yes. Please try the scripts under nn_fwi folder. I think the FWI_forward.jl and FWI_inversion.jl works on both GPU and CPU. I will update other scripts too.
